src/test_utils/public/stub_index_pattern.js

```diff
@@ -37,6 +37,7 @@ export default function (Private) {
     this.title = pattern;
     this.popularizeField = sinon.stub();
     this.timeFieldName = timeField;
+    this.isTimeBased = () => Boolean(timeField);
```

Comment:
__question:__ is there any chance people may want to override return value of this method in the test? If yes, we'd rather write something like this instead: `sinon.stub().returns(Boolean(timeField))`

Reply:
I think they shouldn't since it would be very inconsistent how the real class behaves. Because if they would potentially overwrite it, `this.timeFieldName` would be set, but that method would return `false` which is a scenario that never would happen with the real index pattern class.

Reply:
Okay, I see, then maybe `this.isTimeBased = () => Boolean(this.timeFieldName);` in case someone decides to make `this.timeFieldName = undefined/null`? I'm not sure how exactly it's used, so feel free to ignore :)